pkg/clusterd/disk.go

```diff
@@ -127,11 +127,13 @@ func PopulateDeviceInfo(d string, executor exec.Executor) (*sys.LocalDisk, error
 	}
 
 	// get the UUID for disks
-	var diskUUID string
-	if diskType != sys.PartType {
-		diskUUID, err = sys.GetDiskUUID(d, executor)
+	diskUUID := ""
```

Comment:
if we keep it empty then this new addition is not needed right? It's not a big deal though...